docs/_documentations/mdt-eclipse-usingadifferenttemplate.md

```diff
@@ -10,15 +10,10 @@ order: 8
 parent: usingcodewind
 ---
 
-# Using a template repository of your own
+# Configuring template repositories
```

Comment:
Should be Configuring template sources

Reply:
It should be displaying as such in the latest commit. :)